jekyll-assets/_layouts/boxes.html

```diff
@@ -25,7 +25,7 @@
             {% else %}
             <a class="box" href="{{ entry.url }}">
             {% endif %}
-              <span><img src="{{ site.baseurl }}{{ entry.imagepath }}"></span>
+              <span><img src="{{ site.baseurl }}{{ entry.imagepath }}" width="121px" height="121px" alt=""></span>
```

Comment:
The categories are shown as text, so the images above the text remain only decorative. And with this template no individual alt attribute can be set anyway, otherwise "Linux logo", "Gears", "Raspberry Pi PCB schema" etc would be thinkable for the individual images. Of course a new `entry.imagealt` variable could be implemented, if this was really wanted.

Practically whether it's empty or unset doesn't make a difference, but SEO tools (and also those included in search machines) do not like when the attribute is omitted completely but are fine when it is set to an empty string explicitly for decorative images. So it became best practice.

Reply:
> Of course a new `entry.imagealt` variable could be implemented, if this was really wanted.

@aallan I'm happy to add support for this to `documentation/index.json` if this is something you want.

Reply:
Hmm 🤔 . Probably not at this point.